apps/ideaspace/src/components/common/DialogBox/DialogBoxButton.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { atoms } from '@devlaunchers/components/src/components';
+
+const DialogBoxButton = ({ handleConfirmButton, handleCancelButton, buttonDetail }) => {
+  // when two buttons are same type, first onClick is Confirm, second is Cancel
+  // when only one button, onClick is Cancel.
+  if (buttonDetail[0] == 'alternative primary') {
+    return (
+      <>
+        <atoms.Button
+          buttonSize='standard'
+          buttonType='alternative'
+          onClick={handleCancelButton}
+        >
+          {buttonDetail[1]}
```

Comment:
Is there anyway we could pass in 3 separate objects here, instead of a list of 3 objects? This would help with readability and help to make the code less "brittle"